bundler.d/libvirt.rb

```diff
@@ -1,4 +1,4 @@
 group :libvirt do
-  gem 'fog-libvirt', '>= 0.12.0'
-  gem 'ruby-libvirt', '~> 0.5', :require => 'libvirt'
+  gem 'fog-libvirt', '~> 0.13.0'
```

Comment:
This means that a 0.14.0, once released, will not be considered until we manually change the definition.
Is that the intention here? (I'm OK with that, just making sure the change from `>=` to `~>` is intentional)

Reply:
> will not be considered until we manually change the definition

Yes, that was my intention.  I personally prefer to have stricter limitations in versioning, but if the community thinks otherwise, I'll change it.

Now I see that I have `>=` for `ruby-libvirt`, which I probably should change as well to `~>`

Reply:
I like tight versioning, if there is some automation that regularly bumps those requirements (like dependabot does).
Otherwise, new versions of dependencies might go unnoticed for a longer period of time and you end up having to update for not one bump but multiple. (Granted, we co-maintain fog-libvirt, so it might not be as bad in this exact case).

Sadly dependabot doesn't support our bundler.d setup :(